Library/Formula/fcitx-remote-for-osx.rb

```diff
@@ -0,0 +1,18 @@
+class FcitxRemoteForOsx < Formula
+  desc "handle input method in command-line"
+  homepage "https://github.com/CodeFalling/fcitx-remote-for-osx"
+  url "https://github.com/CodeFalling/fcitx-remote-for-osx/archive/0.0.2.tar.gz"
+  sha256 "746505694090d98d5ad572758e7597c49d3bea1b00208a8272c3648aad15ad69"
+  option "with-input-method=", "Select input method: baidu-pinyin(default), baidu-wubi, sogou-pinyin, qq-wubi, squirrel-rime, osx-pinyin"
```

Comment:
@bfontaine We do in a few places to make it explicit one is the default. I'm fine with just the final `else` personally but some people prefer the explicit option.